rosetta-source/src/main/rosetta/observable-asset-type.rosetta

```diff
@@ -497,3 +497,6 @@ type DividendApplicability: <"The parameters which define whether dividends are
     optionsExchangeDividends boolean (0..1) <"If present and true, then options exchange dividends are applicable.">
     additionalDividends boolean (0..1) <"If present and true, then additional dividends are applicable.">
     allDividends boolean (0..1) <"Represents the European Master Confirmation value of 'All Dividends' which, when applicable, signifies that, for a given Ex-Date, the daily observed Share Price for that day is adjusted (reduced) by the cash dividend and/or the cash value of any non cash dividend per Share (including Extraordinary Dividends) declared by the Issuer. All Dividends in accordance with the ISDA 2002 Equity Derivatives Definitions.">
+    dividendReinvestment cdm.product.asset.DividendPayoutRatio (0..*) <"Specify dividend ratio to be reinvested in notional quantity.">	
+    dividendPassThrough cdm.product.asset.DividendPayoutRatio (0..*) <"Specify dividend ratio to be settled as a stand-alone transfer.">	
```

Comment:
do not understand this problem with "namespace" : 

- a lot of objects are re-used in different namespaces, and this is why most namespaces "impot" other ones, right ? 
- "namespace" eventually is "best effort" classification but per se cannot be an argument to decide where to use a particular compoent... i''m trying not to create new type whenver i can re-use one, and DividendApplicability meet the buiness needs where i'm using it, so are you saying that for "namespace" reason, you would recommand not to use a given type that is relevant ? 

to the point : once you say "namespace" origin is problematic, what do you propose as an alternate modelling ? please help because i do not undertsand the problem with "namespace" and accordingly do not see how to solve it neither